Header header
geometry_msgs/Point32 left_curb
geometry_msgs/Point32 right_curb
int64 left_pt_id
int64 right_pt_id
